embulk-docs/src/developers/index.rst

```diff
@@ -0,0 +1,38 @@
+For core/plugin developers
+===========================
+
+Plugin API documents
+---------------------
+
+* `JavaDoc <javadoc/index.html>`_
+* `RDoc <rdoc/_index.html>`_
+
+Developer guidelines
+---------------------
+
+* Guidelines for core developers (TBA)
+* Guidelines for plugin developers (TBA)
+
+"Embulk v2"
+-----------------------
+
+The Embulk project plans a set of big updates in the Embulk core so that Embulk, its community and the eco-system keep sustainable. Backward compatibility in plugins and configurations is considered especially carefully. Other parts, for example command line options, may have some incompatibility. Join discussions if you have concerns or requests!
+
+JFYI: "Embulk v2" is just a code name. The next versions will be still ``0.?``.
```

Comment:
Agree. Next major version v0.9 is good for Embulk v2 but, we already have v0.9 planning. I think that we need to cleanup/integrate both of them to avoid any confusing. 
https://github.com/embulk/embulk/milestone/1